packages/api/src/domain/__tests__/make-connected-user.ts

```diff
@@ -0,0 +1,20 @@
+import { ConnectedUser } from "../../models/connected-user";
+import { ProviderSource } from "@metriport/api-sdk";
+
+export function makeConnectedUser(
+  userId: string,
+  cxId: string,
+  token: string
+): Partial<ConnectedUser> {
```

Comment:
Can't we return a full `ConnectedUser`? Usually we have `Partial<...>` on the params, so we can choose what to provide to build the object - like you did for `makeUserToken()`.

`Partial<>` on the params also helps to avoid passing unnecessary stuff to build the object, like this from `packages/api/src/providers/__tests__/oauth1-deregister.test.ts`:

```ts
 ... makeConnectedUser(userId, cxId, "")
```

(it adds overhead for 1-2 params, though, having to write the prop's name - so NABD to keep like we have here and just change the return type)

Reply:
Creating a full version of a ConnectedUser, I think, requires calling a ConnectedUser.create() method. Otherwise, the return will be missing some 30+ attributes. I changed the function to receive and also return `Partial<ConnectedUser>`. Lmk if that works, please.

Reply:
Gotcha. Yeah, that's one of the benefits of having a Domain shape separated from the model/DB one - see `Patient` vs. `PatientModel` for reference.

Sounds good for now, at some point we'll likely want to introduce a domain shape to `ConnectedUser` and rename the existing one to `ConnectedUserModel` - maybe you could ticket it as outcome of/compromise for this?